### ISSUE TITLE
[Feature request] Add tooltip on window title in top bar, when titile is long

### ISSUE THREAD

**Issue by eugene-rom** [hardpixel/unite-shell#178]
If long title does not fit, show tooltip with full title text.

![screenshot](https://user-images.githubusercontent.com/33018370/80951635-ea48b880-8e00-11ea-89b3-8d7417c2fa01.png)

**Comment by HubKing** [comment]
Haha. I just came here to suggest almost the same thing. 

Since the monitor is wide, the long title bar area seems to be wasted. So, I integrated the "favourites applications" into the top bar. Most of the time, it is fine, but the problem is that some applications could have a REALLY long title. For example, FireFox shows the page's title on the title bar, and YouTube shows the video's title on the page title. So, a YouTube video with a long title can cause this.

Of course, I could disable the title feature and make it show only the application name (e.g., just "FireFox"), but then the top bar space is wasted, and I cannot read the title text of the application.

How about having a "Maximum title bar length" option? Like a 50 characters or something. Then, if the title is longer than that, it could cut it, with a predefined trim option like "Cut last, Cut middle, or Cut start".

 * Original: This is a stupid YouTube video. Do not watch - YouTube - Mozilla FireFox
 * Cut last: This is a stupid YouTube video. Do not watch - YouTu...
 * Cut middle: This is a stupid YouTube .... YouTube - Mozilla FireFox
 * Cut start: ....video. Do not watch - YouTube - Mozilla FireFox

Showing the full title when the mouse pointer is hovering over it and the title has been cut would be a nice feature to have, but I do not think it is that essential, I think. I do not need to see the full title that often, and if I do, I could just restore the window to see the title.
